app/helpers/application_helper/button/vm_webmks_console.rb

```diff
@@ -7,8 +7,18 @@ def visible?
   end
 
   def disabled?
-    @error_message = _("The web-based WebMKS console is not available because the required libraries aren't installed") unless webmks_assets_provided?
-    @error_message = _('The web-based WebMKS console is not available because the VM is not powered on') unless on?
+    canned_msg = _('The web-based WebMKS console is not available because')
```

Comment:
Please, don't compose senteses via string concatenation. This might not translate to other languates :-(

Replacing the "because" with ":" and starting with a capital letter. Might be a bette choice.

ping @mzazrivec

Reply:
Correct. The message needs to stay together. I get complaints regarding this from translators.